### PR TITLE
Added New DNS_READY

### PR DESCRIPTION
**Pull request by pvasant**
@vkareh PTAL

### CODE REVIEW

**Comment by vkareh** [comment]
Indentation looks off, can you fix it?

File: model/clusters_mgmt/v1/cluster_type.model [hunk 1]
```diff
@@ -96,6 +96,9 @@ class Cluster {
 	// Flag indicating if the cluster is BYOC (customer cloud subscription).
 	BYOC Boolean
 
+    	// Provisioner DNS Ready Status
+    	DNSReady Boolean
```
Comment (vkareh) [inline model/clusters_mgmt/v1/cluster_type.model:100]: This line seems to be using spaces for indentation, should be tabs

Comment (pvasant) [reply model/clusters_mgmt/v1/cluster_type.model:100]: I m not sure why my IDE doesn't show the indentation issue

**Comment by vkareh** [comment]
Thanks! /lgtm

**Pull request merged**